src/components/ArtistProfiles.res

```diff
@@ -181,7 +183,10 @@ module ArtistDetails = {
           | Some(orgList) => <>
               <br />
               <br />
-              <h4> {"Organisations this artist has contributed to:"->React.string} </h4>
+              <h4> {"Organisations that "->React.string} {optName
+                ->Option.getWithDefault(Option.getWithDefault(optArtistName, "Loading artist name"))
+                ->React.string} {" has created artwork for:"->React.string}</h4>
```

Comment:
I kind of like the idea that the artists are "contributing" not just "creating artwork".